### ISSUE TITLE
bug: calcite-icon a11y tests are failing

### ISSUE THREAD

**Issue by pr3tori4n** [Esri/calcite-components#286]
![image](https://user-images.githubusercontent.com/12614215/72112125-8a7a4f00-32f1-11ea-95ba-ccb740c75109.png)

Adding the a11y test helpers while working on pagination component. Found an error in the calcite-icon.

`role='img'` needs alt text apparently. Should this be hardcoded to match the icon name or should this be a prop?

**Comment by jcfranco** [comment]
> Should this be hardcoded to match the icon name or should this be a prop?

The Ionic icon component has a prop and falls back to the icon name. I don't like the fallback option because most icon names aren't very informative. And I don't think we gain a lot from the prop (compared to setting `aria-label` or `aria-labelledby` or `title`).

I'm leaning towards treating the component as an `img` and leaving it up to the user to decide which attributes to use.

**Comment by jcfranco** [comment]
Installed.

**Comment by pr3tori4n** [comment]
Verified
